components/comment.js

```diff
@@ -207,18 +207,31 @@ export default function Comment ({
                 }}
               />
               )
-            : (
-              <div className={styles.text} ref={textRef}>
-                {item.searchText
-                  ? <SearchText text={item.searchText} />
-                  : (
-                    <Text itemId={item.id} topLevel={topLevel} rel={item.rel ?? UNKNOWN_LINK_REL} outlawed={item.outlawed} imgproxyUrls={item.imgproxyUrls}>
-                      {item.outlawed && !me?.privates?.wildWestMode
-                        ? '*stackers have outlawed this. turn on wild west mode in your [settings](/settings) to see outlawed content.*'
-                        : truncate ? truncateString(item.text) : item.text}
-                    </Text>)}
-              </div>
-              )}
+            : href
+              ? (
+                <div ref={textRef}>
+                  {item.searchText
+                    ? <SearchText text={item.searchText} />
+                    : (
+                      <Link className={styles.commentLink} href={href} rel={item.rel ?? UNKNOWN_LINK_REL}>
+                        {item.outlawed && !me?.privates?.wildWestMode
+                          ? '*stackers have outlawed this. turn on wild west mode in your [settings](/settings) to see outlawed content.*'
+                          : truncate ? truncateString(item.text) : item.text}
+                      </Link>)}
+                </div>
```

Comment:
I think I need to reconsider turning the entire comment into a link as I dind't consider links being in a comment therby coming back around to the problem we started with.

![image](https://github.com/stackernews/stacker.news/assets/108441023/d6a0aaeb-1478-40df-bcd1-35c40d5e236d)

Reply:
unfortunately this breaks the comment notifications being visually "read".